go/vt/vtgate/executor_framework_test.go

```diff
@@ -45,359 +46,11 @@ import (
-					"column": "user_id",
-					"name": "hash_index"
-				}
-			]
-		},
-		"music": {
-			"column_vindexes": [
-				{
-					"column": "user_id",
-					"name": "hash_index"
-				},
-				{
-					"column": "id",
-					"name": "music_user_map"
-				}
-			],
-			"auto_increment": {
-				"column": "id",
-				"sequence": "user_seq"
-			}
-		},
-		"music_extra": {
-			"column_vindexes": [
-				{
-					"column": "user_id",
-					"name": "hash_index"
-				},
-				{
-					"column": "music_id",
-					"name": "music_user_map"
-				}
-			]
-		},
-		"music_extra_reversed": {
-			"column_vindexes": [
-				{
-					"column": "music_id",
-					"name": "music_user_map"
-				},
-				{
-					"column": "user_id",
-					"name": "hash_index"
-				}
-			]
-		},
-		"insert_ignore_test": {
-			"column_vindexes": [
-				{
-					"column": "pv",
-					"name": "music_user_map"
-				},
-				{
-					"column": "owned",
-					"name": "insert_ignore_idx"
-				},
-				{
-					"column": "verify",
-					"name": "hash_index"
-				}
-			]
-		},
-		"noauto_table": {
-			"column_vindexes": [
-				{
-					"column": "id",
-					"name": "idx_noauto"
-				}
-			]
-		},
-		"keyrange_table": {
-			"column_vindexes": [
-				{
-					"column": "krcol_unique",
-					"name": "krcol_unique_vdx"
-				},
-				{
-					"column": "krcol",
-					"name": "krcol_vdx"
-				}
-			]
-		},
-		"ksid_table": {
-			"column_vindexes": [
-				{
-					"column": "keyspace_id",
-					"name": "keyspace_id"
-				}
-			]
-		},
-		"t1": {
-      		"column_vindexes": [
-				{
-				  	"column": "id",
-				  	"name": "hash_index"
-				},
-				{
-				  	"column": "unq_col",
-				  	"name": "t1_lkp_vdx"
-				}
-            ]
-    	},
-		"t1_lkp_idx": {
-			"column_vindexes": [
-				{
-					"column": "unq_col",
-				  	"name": "hash_index"
-				}
-			]
-		},
-		"t2_lookup": {
-      		"column_vindexes": [
-				{
-				  	"column": "id",
-				  	"name": "hash_index"
-				},
-				{
-					"column": "wo_lu_col",
-					"name": "t2_wo_lu_vdx"
-				},
-				{
-					"column": "erl_lu_col",
-					"name": "t2_erl_lu_vdx"
-				},
-				{
-					"column": "srl_lu_col",
-					"name": "t2_srl_lu_vdx"
-				},
-				{
-					"column": "nrl_lu_col",
-					"name": "t2_nrl_lu_vdx"
-				},
-				{
-					"column": "nv_lu_col",
-					"name": "t2_nv_lu_vdx"
-				},
-				{
-				  	"column": "lu_col",
-				  	"name": "t2_lu_vdx"
-				}
-            ]
-    	},
-		"user_region": {
-			"column_vindexes": [
-				{
-					"columns": ["cola","colb"],
-					"name": "regional_vdx"
-				}
-			]
-    	},
-		"tbl_cfc": {
-			"column_vindexes": [
-                {
-                    "column": "c1",
-                    "name": "cfc"
-                }
-			],
-			"columns": [
-				{
-					"name": "c2",
-					"type": "VARCHAR"
-				}
-			]
-    	},
-		"zip_detail": {
-			"type": "reference",
-			"source": "TestUnsharded.zip_detail"
-		}
-	}
-}
-`
+//go:embed executorVSchema.json
+var executorVSchema string
+
+//go:embed unshardedVschema.json
+var unshardedVSchema string
```

Comment:
nit: can we move this to a different package like `./vtgate/testdata/` having json in the same package is confusing if it is only used in testing.